pkgs/data/fonts/lucide/default.nix

```
@@ -0,0 +1,33 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+}:
+stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
Nit: `stdenv.mkDerivation` supports a new pattern for recursively accessing its own values. Switching to that pattern will avoid some [common pitfalls regarding recursive attribute sets](https://nix.dev/guides/best-practices#recursive-attribute-set-rec). You'll then be able to access this derivation's values by prefixing them with `finalAttrs`, like this:
```nix
{
  tag = "v${finalAttrs.version}"
}
```
Don't forget the closing parenthesis at the end of the file.